editor/src/core/performance/performance-regression-tests.spec.tsx

```diff
@@ -116,7 +116,7 @@ describe('React Render Count Tests - ', () => {
     )
 
     const renderCountAfter = renderResult.getNumberOfRenders()
-    expect(renderCountAfter - renderCountBefore).toBeGreaterThanOrEqual(630) // if this breaks, GREAT NEWS but update the test please :)
-    expect(renderCountAfter - renderCountBefore).toBeLessThan(640)
+    expect(renderCountAfter - renderCountBefore).toBeGreaterThanOrEqual(740) // if this breaks, GREAT NEWS but update the test please :)
```

Comment:
That seems surprising.

Reply:
I didn't yet spend time to verify, but I think what happens is that we render an inspector before the DOM walker metadata is merged in, and then we render a second time when it does arrive. and I changed the inspector to hide all values if the DOM metadata contains a null for computedStyle, which means we render an empty inspector first, and a full inspector second. If that is indeed the case, then I will change the inspector to render nothing for an empty computed style. 

there's an opportunity to use React Suspense here, and literally not render an inspector until we have the data ready, I'll see if I can use that API for our benefit

Reply:
Ooooh ok that would absolutely make sense, since the DOM walker doesn't run until after the first Canvas render